hooks/21-snappy-security-policy-stamp.chroot

```diff
@@ -1,18 +0,0 @@
-#!/bin/sh
```

Comment:
Grepping for `security-policy-version` on the entire image left no hits so I assume this is a 15.04 leftover that can be removed.